aiosmtpd/tests/test_smtp.py

```diff
@@ -7,8 +7,15 @@
 
 from aiosmtpd.controller import Controller
 from aiosmtpd.handlers import Sink
-from aiosmtpd.smtp import SMTP as Server, __ident__ as GREETING
-from aiosmtpd.testing.helpers import reset_connection
+from aiosmtpd.smtp import MISSING, SMTP as Server, __ident__ as GREETING
+from aiosmtpd.testing.helpers import (
+    SUPPORTED_COMMANDS_NOTLS,
+    assert_auth_invalid,
+    assert_auth_required,
+    assert_auth_success,
+    reset_connection,
+)
```

Comment:
These are centralized because they are often used, and if the output ever changes it will be much easier to change the expected output in one place rather than searching this honking big file for all instances of same.